android/assets/jsons/translations/German.properties

```diff
@@ -2892,8 +2878,7 @@ Incompatible with [param] = Inkompatibel mit [param]
 Philantropy = Philantropie
 Gifts of Gold to City-States generate [amount]% more Influence = Goldgeschenke an Stadtstaaten erzeugen [amount]% mehr Einfluss
 Aesthetics = Ästhetik
- # Requires translation!
-Resting point for Influence with City-States is increased by [amount] = 
+Resting point for Influence with City-States is increased by [amount] = Basispunkt für Einfluss mit Stadtstaaten wird um [amount] erhöht
```

Comment:
@SomeTroglodyte Da ich Civ5 nie gespielt habe, finde ich nicht die richtige Übersetzung für "Resting point". Hast du eine gute Idee?

Reply:
Ehrlich gesagt, ich hab's auch noch nich' richtig in Aktion beobachtet... Muß eigentlich so sein = man kauft 30 Punkte, das geht dann jede Runde um 1 runter zur Null. Dieser Effekt verschiebt den Nullpunkt wo es hin "altert"? Bestimmt. 

Versteht man "Ruhepunkt"? Wäre ja fast wörtlich. Sagt auch [g00 :vomiting_face: gle](https://translate.google.com/?sl=en&tl=de&text=Resting%20point&op=translate). Oder [leo](https://dict.leo.org/forum/viewUnsolvedquery.php?idThread=1120884&idForum=2&lang=en&lp=ende) - dort auch Vorschlag "Ruhezustand"...

Reply:
Was hälst du von "Ausgangspunkt"?